libgo_owasm/build.rs

```diff
@@ -0,0 +1,9 @@
+use std::env;
+
+fn main() {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+
+    cbindgen::generate(crate_dir)
+        .expect("Unable to generate bindings")
+        .write_to_file("../api/bindings.h");
+}
```

Comment:
What does it do

Reply:
to generate an interface file (bindings.h) for go (cgo)

Reply:
why it expect to unable to generate?

Reply:
It's an error message ja